package.json

```diff
@@ -753,11 +767,21 @@
     "azure-iot-device": "^1.8.1",
     "azure-iot-device-mqtt": "^1.8.1",
     "azure-iothub": "^1.8.1",
+    "body-parser": "^1.18.2",
+    "dotenv": "^5.0.1",
+    "dummy-json": "^2.0.0",
+    "express": "^4.16.3",
     "fs-extra": "^7.0.0",
+    "is-port-reachable": "^2.0.1",
     "ms-rest": "^2.3.2",
     "ms-rest-azure": "^2.5.5",
+    "net": "^1.0.2",
     "replace-in-file": "^3.4.0",
+    "request": "^2.88.0",
+    "request-promise": "^4.2.2",
+    "semver": "^5.6.0",
     "strip-json-comments": "^2.0.1",
+    "ts-node": "^8.3.0",
```

Comment:
remove unused dependencies

Reply:
May I ask how do I know which ones are not being used?